docs/NOW-spec.md

```diff
@@ -0,0 +1,1601 @@
+
+# NOW-PROTO
+
+<style>
+    .byte-layout {
+        width: 100%;
+        table-layout: fixed;
+    }
+    .byte-layout th {
+        colspan: 1;
+        text-align: center;
+        vertical-align: bottom;
+    }
+    .byte-layout td {
+        text-align: center;
+    }
+</style>
```

Comment:
Does this actually takes effects? Not sure if GitHub allows styling.

![image](https://github.com/user-attachments/assets/f75aa076-b1d9-4785-95bc-4b6aa24aca5b)

Or are you viewing from somewhere else?

Reply:
Yep, It should be viewed in Markdown Monster, @awakecoding has written initial document here (see [this message](  https://devolutions.slack.com/archives/GVCJYNXBJ/p1718913383781139?thread_ts=1718744469.277199&cid=GVCJYNXBJ))